seed/php-sdk/pagination/no-custom-config/README.md

```diff
@@ -68,6 +68,41 @@ try {
 }
 ```
 
+## Pagination
+
+List endpoints return a `Pager<T>` which lets you loop over all items and the SDK will automatically make multiple HTTP requests for you.
+
+```php
+<?php
+
+namespace Example;
+
+use Seed\SeedClient;
+use Seed\Complex\Types\SearchRequest;
+use Seed\Complex\Types\StartingAfterPaging;
+use Seed\Complex\Types\SingleFilterSearchRequest;
+use Seed\Complex\Types\SingleFilterSearchRequestOperator;
+
+$client = new SeedClient(
+    token: '<token>',
+);
+$client->complex->search(
+    'index',
+    new SearchRequest([
+        'pagination' => new StartingAfterPaging([
+            'perPage' => 1,
+            'startingAfter' => 'starting_after',
+        ]),
+        'query' => new SingleFilterSearchRequest([
+            'field' => 'field',
+            'operator' => SingleFilterSearchRequestOperator::Equals->value,
+            'value' => 'value',
+        ]),
+    ]),
+);
+
+```
+
```

Comment:
<img width="1108" alt="Screenshot 2025-06-26 at 8 00 40 PM" src="https://github.com/user-attachments/assets/cbf72f32-d54c-4cd8-b34f-7a084c408f01" />
 we also need to add this section